web/app/scripts/src/services/api.es.js

```diff
@@ -33,6 +33,12 @@ module.service(moduleProperties.componentName, [
             getForm: function (url) {
                 return $http.get(url)
                     .then((response) => response.data);
+            },
+            post: function (baseUrl, data) {
+                const endpoint = 'respond';
```

Comment:
I don't think it's such a great solution. Manually setting endpoints is not great, either way. The real solution would be to get the list of endpoints from the backend. But we can fix that later.